include/uprotocol-cpp/cloudevent/factory/cloud_event_factory.h

```diff
@@ -469,7 +469,7 @@ struct factory {
       (*(ce).mutable_attributes())[Serializer::REQ_ID_KEY] = *attr;
     }
 
-    if (unlikely(rpcUri.empty() || !valid_uri(rpcUri))) {
+    if (unlikely(rpcUri.empty() || !UriValidator::isValidUri(rpcUri))) {
```

Comment:
We should not be using any C++20 constructs.  Remove all "likely" and "unlikely"'s.

Reply:
Done